package.json

```diff
@@ -31,7 +31,7 @@
     "prop-types": "^15.5.10",
     "query-string": "^5.1.1",
     "react": "^16.4.2",
-    "react-beautiful-dnd": "^10.0.4",
+    "react-beautiful-dnd": "^10.1.0",
```

Comment:
Was this actually necessary or just something you were in there for anyway?

Reply:
This was a recent update and we might as well pull it in.